Gems/AWSGameLift/Code/AWSGameLiftCommon/Source/AWSGameLiftSessionConstants.h

```diff
@@ -10,11 +10,13 @@
 
 namespace AWSGameLift
 {
+    AZ_PUSH_DISABLE_WARNING_GCC("-Wunused-variable")
     // Reference https://sdk.amazonaws.com/cpp/api/LATEST/_game_session_status_8h_source.html
     static const char* AWSGameLiftSessionStatusNames[6] = { "NotSet", "Active", "Activating", "Terminated", "Terminating", "Error"};
 
     // Reference https://sdk.amazonaws.com/cpp/api/LATEST/_game_session_status_reason_8h.html
     static const char* AWSGameLiftSessionStatusReasons[2] = { "NotSet", "Interrupted" };
+    AZ_POP_DISABLE_WARNING_GCC
```

Comment:
minor: For these 2 variables maybe it'd be easier just to add `[[maybe_unused]]` than surrounding with macros.

Reply:
constexpr works as well. I dont know why I didnt do that earlier